### ISSUE TITLE
(>3.6.5) opening links in terminal instantly kills the window

### ISSUE THREAD

**Issue by noteventhrice** [mintty/wsltty#346]
Ctrl-clicking links in the terminal closes it instantly and does not open the link in any version above 3.6.5, downgrading back to 3.6.5 and adding in latest wslbridge2 solved this for me, but it would be nice to be able to update

### environment info:
```
WSL version: 2.0.9.0
Kernel version: 5.15.133.1-1
WSLg version: 1.0.59
MSRDC version: 1.2.4677
Direct3D version: 1.611.1-81528511
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.19045.3693
```
### wsl distro and type:
```
  NAME            STATE           VERSION
* Ubuntu-20.04    Running         2
```

### Abridged Config:
```
Term=xterm-256color
Scrollbar=none
ClicksTargetApp=no
Locale=en_US
Charset=UTF-8
MiddleClickAction=void
Language=en_US
BoldAsColour=no
Transparency=off
OpaqueWhenFocused=yes
FontSmoothing=default
Charwidth=unicode
CursorType=block
Emojis=apple
EmojiPlacement=middle
FontWeight=400
ScrollbackLines=3000
HandleDPI=2
FontIsBold=no
CopyAsRTF=no
Columns=96
Rows=24
StatusLine=no
```

**Comment by noteventhrice** [comment]
After viewing the diff I believe this is an issue with mintty.

**Comment by noteventhrice** [comment]
Built wsltty and mintty from master, confirmed that clicking links no longer crashes the term and opens the link. Thanks!

<img width="401" alt="oix2z8zo" src="https://github.com/mintty/wsltty/assets/7583394/947eb3b2-916d-4ee4-97d2-c077ab1149fc">

**Comment by mintty** [comment]
Released 3.7.1.
